css/_polls.scss

```diff
@@ -0,0 +1,62 @@
+.pollCreateFormContainer, .voteFormContainer, .pollResultsFormContainer {
+    width: 500px;
+    text-align:center;
+}
+.pollChoicesListContainer {
+    text-align: center;
```

Comment:
These centerings can be removed by adding align-items center to the form container instead, causing all of its children to be centered. Then one parent becomes responsible for alignment instead of multiple.

I should note that the design of the feature will need to be changed. I'm trying to get some commitment from the team to either offer redesigns, which you would do or someone on the team would do in the future, or to allow tweaks to the current design with the feature remaining hidden until designs are received. There are probably many other options other than those two.